### DISCUSSION TITLE
Welcome to draft-ietf-rats-corim Discussions!

### DISCUSSION THREAD

**Discussion by nedmsmith** [ietf-rats-wg/draft-ietf-rats-corim#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nedmsmith** [comment]
Enabling the discussions feature. Nothing to discuss at the moment.

**Reply by thomas-fossati** [reply]
Nice!  Should we publish here the details of our regular sync-up?

**Comment by nedmsmith** [comment]
That would be a good use. Some of the ‘notes’ of the sync meetings are captured in issues, but not all.

**Reply by thomas-fossati** [reply]
see https://github.com/ietf-rats-wg/draft-ietf-rats-corim/discussions/38
